packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCardPanel.tsx

```diff
@@ -74,10 +89,19 @@ export class AddCardPanel extends BaseComponent<IAddCardPanelProps, IAddCardPane
     });
     return (
       <div className={classNames.contentRoot}>
-        <div className={classNames.header} tabIndex={0} aria-label={header}>
-          {header}
-        </div>
-        {addCardItemsList}
+        {addCardItems.length > 0 ? (
+          <>
+            <div className={classNames.header} tabIndex={0} aria-label={header}>
+              {header}
+            </div>
+            {addCardItemsList}
+          </>
+        ) : (
+          <div className={classNames.emptyCardStyles}>
+            <img draggable={false} src={this._getEmptyCard()} />
```

Comment:
You may want to consider providing an `alt` attribute for assistive technology. Its a best-practice and its absence will report ARIA validation errors.

https://www.w3.org/TR/WCAG20-TECHS/H37.html